src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Managing Several Displays with the Multi-Screen Window Placement API
+subhead: Get information about connected displays and position windows relative to those displays.
+authors:
+  - thomassteiner
+description:
+  The Multi-Screen Window Placement API allows you to enumerate the displays connected to your
+  machine and to place windows on specific screens.
+date: 2020-09-03
+# updated: 2020-09-03
+tags:
+  - blog
+  - capabilities
+hero: hero.jpg
+alt: Simulated trading desk showing multiple fake cryptocurrencies and their price charts.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/1411878483180650497
+---
+
+{% Aside %} The Multi-Screen Window Placement API is part of the
+[capabilities project](https://developers.google.com/web/updates/capabilities) and is currently in
+development. This post will be updated as the implementation progresses. {% endAside %}
+
+## What is the Multi-Screen Window Placement API? {: #what }
+
+The Multi-Screen Window Placement API is a new web platform API that allows you to enumerate the
+displays connected to your machine and to place windows on specific screens.
+
+### Suggested use cases for the Multi-Screen Window Placement API {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Multi-window graphics editors à la
+  [Gimp](https://www.gimp.org/release-notes/gimp-2.8.html#single-window-mode) can place various
+  editing tools in accurately positioned windows.
+- Virtual trading desks can show market trends in multiple windows that all can also be viewed in
+  fullscreen mode.
+- Slideshow apps can show speaker notes on the internal primary screen and the presentation on an
+  external projector.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | [Complete][spec]         |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| **4. Origin trial**                      | **[In progress][ot]**    |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use the Multi-Screen Window Placement API {: #use }
+
+### Enabling via chrome://flags
+
+To experiment with the Multi-Screen Window Placement API locally, without an origin trial token,
```

Comment:
Quick question. Does it work on Android or is it "working" only for desktop?

Reply:
I was surprised, too, but this will work [[e]verywhere except WebView](https://groups.google.com/a/chromium.org/g/blink-dev/c/C6xw8i1ZIdE/m/TJsr0zXxBwAJ#:~:text=The%20overall%20API%20will%20be%20exposed%20everywhere%20except%20WebView).